src/core/source_buffers/index.ts

```diff
@@ -28,12 +35,24 @@ import SourceBuffersStore, {
 
 export default SourceBuffersStore;
 export {
+  QueuedSourceBuffer,
   BufferGarbageCollector,
+
```

Comment:
Is this new way to format exports ?

Reply:
There was too much types here.

The empty lines are attemps to make it more readable by grouping related types together.